tests/conftest.py

```diff
@@ -10,6 +10,19 @@
 # Ensure progress output even when not outputting to raw sys.stderr console
 os.environ["DVC_IGNORE_ISATTY"] = "true"
 
+REMOTES = {
+    # remote: enabled_by_default?
+    "azure": False,
+    "gdrive": False,
+    "gs": False,
+    "hdfs": False,
+    "http": True,
```

Comment:
The `True` remotes will always run by default. To disable them, you can use `--disable-http` for example.